torch/onnx/experimental/fonnx/v1/exporter.py

```diff
@@ -0,0 +1,116 @@
+import dataclasses
+import io
+from enum import Enum
+from typing import Optional, Protocol, Union
+
+import torch
+
+
+def dynamo_export(
+    model: torch.nn.Module,
+    /,
+    *model_args,
+    export_options: Optional[ExportOptions] = None,
+    **model_kwargs,
+) -> ExportOutput:
+    """Export a torch.nn.Module to a ONNX graph
+
+    Args:
+        model: Model to be exported
+        model_args: Positional inputs to `torch_model`
+        export_options: Export options
+        model_kwargs: Keyword inputs to `torch_model`
+
+    Returns:
+        An in-memory representation of the exported ONNX model.
+        See ``ExportOutput`` for reference.
+    """
+    ...
+
+
+class ExportLogLevel(Enum):
+    """Class for choosing log level.
+
+    Names and values follows https://docs.python.org/3/library/logging.html#levels"""
+
+    CRITICAL = 50
+    ERROR = 40
+    WARNING = 30
+    INFO = 20
+    DEBUG = 10
+    NOTSET = 0
+
+
+@dataclasses.dataclass
+class ExportOptions:
+    """Class for keeping track of ONNX exporter flags can perform."""
+
+    opset_version: Optional[int] = None
+    """opset_version: Target ONNX opset version
+
+    When not specified, the exporter selects the latest supported version."""
+
+    _log_level: Optional[Union["ExportLogLevel", str, int]] = None
+    """Log level as defined by https://docs.python.org/3/library/logging.html#levels"""
+
+    @property
+    def log_level(self) -> ExportLogLevel:
```

Comment:
no it doesn't :( https://stackoverflow.com/a/61480946

Reply:
meanwhile, do we want to set a static default level?

Reply:
* reconciled `log_level` property on the dataclass as described by Bowen's reference (thanks!)
* @abock suggestion was to never set a default and let exporter silently pick for the user when they don't specify. if they want a specific behavior, they set it themselves. 
  * The pro is that we never break public interface, because it never changes. 
  * The con is that the user does not know what is the default.
  * Personally, I like knowing default values from the public interface, but it is not a strong preference